pom.xml

```diff
@@ -51,7 +51,7 @@
     <version.io.swagger>1.5.3</version.io.swagger>
     <version.org.apache.commons-commons-math3>3.6</version.org.apache.commons-commons-math3>
     <version.org.apache.commons-commons-lang3>3.4</version.org.apache.commons-commons-lang3>
-    <version.com.squareup.okhttp>2.7.4</version.com.squareup.okhttp>
+    <version.com.squareup.okhttp>3.4.2</version.com.squareup.okhttp> <!-- should be picked up from parent pom -->
```

Comment:
This would require a bump of the parent-pom to the latest, but I noticed this is using a very old parent pom of version 39 (see https://github.com/hawkular/hawkular-datamining/blob/master/pom.xml#L25 ) I wasn't sure if there was a reason why this wasn't upgraded recently, so I left it. I can certainly do this, but would need a +1 from you @pavolloffay  to say if that is ok to move up the parent pom that much (from 39 all the way to 51)